indico/modules/events/layout/forms.py

```diff
@@ -39,7 +39,33 @@ def _get_conference_theme_choices():
     return THEMES + sorted(plugin_themes, key=lambda x: x[1].lower())
 
 
-class ConferenceLayoutForm(IndicoForm):
+class LoggedLayoutForm(IndicoForm):
+    @classmethod
+    def build_field_metadata(cls, field):
+        if field.short_name == 'name_format':
+            return {'title': orig_string(field.label.text),
+                    'type': 'string',
```

Comment:
For this particular one the enum type should be fine, shouldn't it?

Reply:
Yep, but I had to fix the `Enum` interpretation in `make_diff_log`.